solarwinds_apm/propagator.py

```diff
@@ -121,22 +121,32 @@ def inject(
             carrier, self._TRACESTATE_HEADER_NAME, trace_state.to_header()
         )
 
-        # Remove any baggage stored for custom transaction naming
-        if baggage_header:
-            baggage_header = self.remove_custom_naming_baggage_header(
-                baggage_header,
+        def handle_baggage_header(baggage_header):
```

Comment:
Do we want to add a leading underscore to indicate it is private to the function?

Reply:
Addressed in [a41b577](https://github.com/solarwinds/apm-python/pull/539/commits/a41b5771c54ab2d7c22733247adc5554b8a1d1b8)